enterprise/backend/src/metabase_enterprise/database_replication/api.clj

```diff
@@ -70,7 +70,7 @@
           ;; apply filter over the memoized result for snappy UI
           hm-preview-memo
           :tables
-          (filter (comp (schema-filters->fn replication-schema-filters) :table_schema))))
```

Comment:
These keys will be looked up properly, and in a backwards and forwards compatible way, because `hm-client.call` now converts them to kebab-case.